engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -100,55 +105,28 @@ public void initClient() throws RemoteComponentException {
     public void closeClient() {
         logger.debug("closing '{}' @{}", componentDisplayName, this.hashCode());
         client = null;
-        // TODO: client now misses a "close" method (although Jersey client has a "destroy" method).
+    }
+
+    @Validate
+    public void validate() throws Exception {
+        CreateInput createInput = new CreateInput();
+        createInput.configuration = getConfiguration(getUsedInputColumns());
+        try {
+            cachedOutputColumns.getCachedValue(createInput);
+        } catch(RESTClientException e) {
+            if(e.getCode() == 422) {
```

Comment:
Without a better alternative I vote to take the 422 that @jakubneubauer suggests :-)